lib/measures/tbd/resources/utils.rb

```diff
@@ -1729,17 +1732,17 @@ def surface_valid?(s = nil)
   # @param s [OpenStudio::Model::Surface] a model surface
   # @param subs [Array] requested sub surface attributes
   # @param clear [Bool] remove current sub surfaces if true
+  # @param bfr [Double] safety buffer (m), when ~aligned along other edges
   #
   # @return [Bool] true if successful (check for logged messages if failures)
-  def addSubs(model = nil, s = nil, subs = [], clear = false)
+  def addSubs(model = nil, s = nil, subs = [], clear = false, bfr = 0.005)
```

Comment:
This is a new OSut [feature](https://github.com/rd2/osut/pull/11), also accessible via TBD.